### DISCUSSION TITLE
Plugin system

### DISCUSSION THREAD

**Discussion by FlamesX-128** [FlamesX-128/anigo#9] [Ideas]
# Plugin system 

## Plugin types.
- [ ~ ] Specialists
- [ ~ ] Controller
- [] Thread
- [] Data
- [] Unknown

## Multilanguage support
- [ ~ ] Rust
- [] C

## Support
- [~] High flexibility
- [~] Settings system

## Contributing
everything is welcome.

**Comment by bachig26** [comment]
will there possible support for fzf or similar functionality?

**Reply by FlamesX-128** [reply]
I don't think it's the best for a cli of this type.

**Reply by FlamesX-128** [reply]
I want the plugin system to be quite flexible, so someone could do that

**Reply by bachig26** [reply]
Okay. 👍
